docs/endpoints/post-token-generate.md

```diff
@@ -69,24 +71,25 @@ The following are unencrypted JSON request body examples for each parameter, one
     "optout_check": 1
 }
 ```
-
-Here's an encrypted token generation request format with placeholder values:
-
-```sh
-echo '[Unencrypted-JSON-Request-Body]' \
-  | encrypt_request.py [CLIENT_SECRET] \
-  | curl -X POST 'https://prod.euid.eu/v2/token/generate' -H 'Authorization: Bearer [CLIENT_API_KEY]' -d @- \
-  | decrypt_response.py [CLIENT_SECRET]
+```json
+{
+    "phone": "+12345678901",
+    "optout_check": 1
+}
+```
+```json
+{
+    "phone_hash": "wdN1alhrbw1Bmz49GzKGdPvGxLhCNn7n3teAOQ/FSK4=",
+    "optout_check": 1
+}
 ```
 
 Here's an encrypted token generation request example for an email hash:
 
 ```sh
-echo '{"email_hash": "tMmiiTI7IaAcPpQPFQ65uMVCWH8av9jw4cwf/F5HVRQ="}' \
-  | encrypt_request.py DELPabG/hsJsZk4Xm9Xr10Wb8qoKarg4ochUdY9e+Ow= \
-  | curl -X POST 'https://prod.euid.eu/v2/token/generate' -H 'Authorization: Bearer YourTokenBV3tua4BXNw+HVUFpxLlGy8nWN6mtgMlIk=' -d @- \
-  | decrypt_response.py DELPabG/hsJsZk4Xm9Xr10Wb8qoKarg4ochUdY9e+Ow=
+echo '{"email_hash": "tMmiiTI7IaAcPpQPFQ65uMVCWH8av9jw4cwf/F5HVRQ=","optout_check":1}' | python3 uid2_request.py https://prod.euid.eu/v2/token/generate [Your-Client-API-Key] [Your-Client-Secret] 
```

Comment:
maybe as mentioned above - replicate [Your-Client-API-Key] [Your-Client-Secret]  for identity/map endpoint doc too :)

Reply:
Yes. Done, thank you. In both.